src/linear_algebra/determinant.lean

```diff
@@ -243,7 +244,8 @@ by { haveI := classical.dec_eq ι,
      assumption }
 
 /-- In a finite-dimensional vector space, the zero map has determinant `1` in dimension `0`,
-and `0` otherwise. -/
+and `0` otherwise. We give a formula that also works in infinite dimension, where the determinant
+is `1`. -/
```

Comment:
Is this generally accepted as the right definition of determinant there, or is that a choice being made by mathlib? If the latter
```suggestion
and `0` otherwise. We give a formula that also works in infinite dimension, where we define
the determinant to be `1`. -/
```